File: BucketList/app/src/main/java/com/example/bucketlist/MainActivity.java
package com.example.bucketlist;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    private CardView cardViewThings;
    private CardView cardViewPlaces;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        cardViewThings = findViewById(R.id.card_view_things);
        cardViewPlaces = findViewById(R.id.card_view_places);

        cardViewThings.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent thingsActivityIntent = new Intent(MainActivity.this, ThingsActivity.class);
                startActivity(thingsActivityIntent);
            }
        });
        cardViewPlaces.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent placesActivityIntent = new Intent(MainActivity.this, PlacesActivity.class);
                startActivity(placesActivityIntent);
            }
        });
    }
}

File: SpartApp/app/src/main/java/com/example/spartapp/MainActivity.kt
package com.example.spartapp

import android.content.Intent
import android.os.Bundle
import android.widget.Toast
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.Button
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.platform.LocalContext
import androidx.compose.ui.tooling.preview.Preview
import com.example.spartapp.ui.theme.SpartAppTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            GoToWorkoutActivityButton()
        }
    }
}

@Composable
fun GoToWorkoutActivityButton() {
    val context = LocalContext.current
    Button(onClick = {
        context.startActivity(Intent(context, WorkoutActivity::class.java))
    }) {
        Text(text = "Workout")
    }
}

@Preview(showBackground = true)
@Composable
fun GreetingPreview() {

}